src/components/aggregates/__test__/AggregatesTable.test.tsx

```diff
@@ -49,10 +55,26 @@ describe("AggregatesTable Component", () => {
         jest.resetAllMocks();
         jest.clearAllMocks();
         jest.restoreAllMocks();
+        result = undefined;
     });
 
-    it('renders aggregates table', async () => {
-        result = renderWithProviders(<ConfirmationProvider><AggregatesTable /></ConfirmationProvider>);
+    it('renders aggregates table with write permission', async () => {
```

Comment:
Please include in the verifications made on the tests with write / read permission, the differences in what is rendered depending on such permissions. Namely, the presence or absence of the delete button in one row.